vsdownload.py

```diff
@@ -640,6 +640,9 @@ def unpackWin10WDK(src, dest):
 
 def extractPackages(selected, cache, dest):
     makedirs(dest)
+    # The path name casing is not consistent across packages, or even within a single package.
+    # Manually create top-level folders before extracting packages to ensure the desired casing.
+    makedirs(os.path.join(dest, "MSBuild"))
```

Comment:
How does this help - if unpacking on a case sensitive file system? Does `msiextract` do a case insensitive comparison with existing directories before creating a new one?